.cirrus.yaml

```diff
@@ -1,5 +1,22 @@
+# The default 2 CPUs / 4G memory is not required.
+container:
+  cpu: 1
+  memory: 2G
+
 unix_bundle_cache: &unix_bundle_cache
   bundle_cache:
+    folder: /usr/local/bundle
```

Comment:
But why do you suggest to use absolute path for Linux if relative has no problems in other projects? Like, again, [`sinatra-r18n`](https://github.com/r18n/sinatra-r18n): https://cirrus-ci.com/task/6211249500323840

Reply:
Yeah, also not 100% sure about that. Will close PR then

Reply:
> Yeah, also not 100% sure about that. Will close PR then

As I wrote, I'm thinking the issue in the CI itself, something has been broken there with cache. And I'm ready to try, to help, to investigate — just write me anywhere. I want to help not to me, but to us: make this awesome CI better and more stable. ❤️